package.json

```diff
@@ -91,6 +91,7 @@
     "classnames": "^2.2.6",
     "clipboard": "^2.0.6",
     "connected-react-router": "^6.9.1",
+    "cypress-browser-permissions": "^1.1.0",
```

Comment:
This should be a `devDependency`. As a `dependency` this would end up in the production bundle.